.devcontainer/post_create_command.sh

```diff
@@ -14,4 +14,8 @@ pip3 install -r test/requirements.txt
 
 cd ..
 
+# Setup cypress
+sudo apt-get install -y libgtk2.0-0 libgtk-3-0 libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb libgbm-dev
```

Comment:
we should not add cypress this way. We should add it in the features section above. An easy way to do this is to go to vscode and do CMD + SHFT + P and type `Configure Container Features` where you can search for Cypress.

also, if the package.json already includes cypress, we dont need to `npm install cypress` since the other npm installs would install it to node_modules.

Reply:
> We should add it in the features section above.
I meant to say that we should add it in the `devcontainer.json`. Thats where we control what things are installed in the dev container. Post_create_command is just something we run after the container has been built but we can set it up so that the container already has cypress installed rather than installing it ourselves

Reply:
I wasn't able to use the features. The only method that worked was post_create_commands with some special tweaks